src/components/CountrySelect.vue

```diff
@@ -66,10 +67,14 @@ export default {
       type: Boolean,
       default: false
     },
+    disablePlaceholder: {
+      type: Boolean,
+      default: false,
+    },
   },
   methods: {
     handleInput(value) {
-      this.$emit('input', value);
+        this.$emit('input', value);
```

Comment:
This is nitpicky, but can you bump this back to how it was?

Reply:
oops... missed that, I had it in an if statement temporarily when trying to find a way to block selection of the placeholder.
Fixed!